### PR TITLE
pro-notes: 0.7.6 (new cask)

### PR DESCRIPTION
**Pull request by daeho-ro**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/p/pro-notes.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+cask "pro-notes" do
+  version "0.7.6"
+  sha256 "655cd62f286539610c0523fee835ac5f3dcccd74bb9d63e245ee51f5a898e8f5"
+
+  url "https://assets.pronotes.app/downloads/ProNotes-#{version}.zip"
+  name "pro-notes"
```
Comment (bevanjkay) [inline Casks/p/pro-notes.rb:6]:
```suggestion
  name "ProNotes"

File: Casks/p/pro-notes.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  url "https://assets.pronotes.app/downloads/ProNotes-#{version}.zip"
+  name "ProNotes"
+  desc "Apple Notes extension"
+  homepage "https://www.pronotes.app/"
+
+  livecheck do
+    url "https://www.pronotes.app/direct-download"
+    strategy :header_match
+  end
```
Comment (bevanjkay) [inline Casks/p/pro-notes.rb:13]:
```suggestion
  livecheck do
    url "https://www.pronotes.app/appcast.xml"
    strategy :sparkle, &:short_version
  end
```

Can you please also add `auto_updates true`

Comment (daeho-ro) [reply Casks/p/pro-notes.rb:13]: How do you find this info? I am just curious.

Comment (daeho-ro) [reply Casks/p/pro-notes.rb]: Updated!

Comment (bevanjkay) [reply Casks/p/pro-notes.rb]: I used an app called proxyman (`brew install proxyman`) - which allows you to intercept the network calls.

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**